### PR TITLE
set up templates for more elements, add sg to all classes

### PR DESCRIPTION
**Pull request by hippogriffic**
make sure styles will only affect style guide page

image

again pls ignore the compiled template, it's the one with all the ugly whitespace :)

@incuna/frontend please review

### CODE REVIEW

File: sass/globals/_colors.sass [hunk 1]
```diff
@@ -1,19 +1,19 @@
-$black: #000
-$white: #fff
+$black: #000 !default
+$white: #fff !default
```
Comment (maciejpi) [inline sass/globals/_colors.sass]: shouldn't black and white always stay as they are defined here?

File: sass/_imports.sass [hunk 2]
```diff
@@ -10,7 +10,7 @@ $prefix-for-opera: false
 // Incuna-sass
 @import incuna-sass/functions
 @import incuna-sass/normalize
-// @import incuna-sass/reset
+@import incuna-sass/incuna-normalize
```
Comment (maciejpi) [inline sass/_imports.sass:13]: aren't we always using both reset and normalize? 

and what's incuna-normalize?

Comment (hippogriffic) [inline sass/_imports.sass:13]: reset didn't offer any real benefit here, it'll likely be added to individual projects anyway so it's not essential to add stuff here.

Incuna normalize is a new file we made with sass we tend to add to every project, so we don't always have to repeat ourselves, things like `box-sizing: border-box`

File: sass/globals/_colors.sass (hunk 1 above)
Comment (hippogriffic) [inline sass/globals/_colors.sass]: yeah you're probably right, I was adding default to everything to make it easily overwritable, this library's sass shouldn't be affecting projects, but it's probably safe to leave these without the default

**Comment by maciejpi** [comment]
HWTI 👍

**Pull request merged**